blivetgui/dialogs/add_dialog.py

```diff
@@ -834,6 +834,8 @@ def update_fs_chooser(self):
         supported_fs = supported_filesystems()
         if self.selected_free.size > size.Size("8 MiB"):
             supported_fs.append("lvmpv")
+        if self.selected_free.size > size.Size("256 MiB"):
```

Comment:
And the ``8 MiB`` should be added as a constant to either blivet or somewhere here in blivet-gui. Why is it ``8 MiB`` anyway?